dags/openshift_nightlies/util/var_loader.py

```diff
@@ -15,19 +15,15 @@ def get_git_user():
     git_user = git_path.split('/')[0]
     return git_user.lower()
 
-def get_secret(name, deserialize_json=False):
-    overrides = get_overrides()
-    if name in overrides:
-        return overrides[name]
-    return Variable.get(name, deserialize_json=deserialize_json)
 
-
-def get_overrides():
-    try:
-        return Variable.get("overrides", deserialize_json=True)
-    except KeyError: 
-        return {}
-    
+def get_secret(name, deserialize_json=False, required=True):
+    if required:
+        return Variable.get(name, deserialize_json=deserialize_json)
+    else:
+        try:
+            return Variable.get(name, deserialize_json=deserialize_json)
+        except KeyError:
+            return {}
 
```

Comment:
why use if else above^? won't just try except be enough?

Reply:
The task variables are not required by default, so when required is False and the variable is not found in vault we return an empty dir

Reply:
It works, but I wanted to add the required logic to this method, so if one of the variables required is not present, the method will hang and not return an empty dir (which can be confusing)